meta-webserver/recipes-webadmin/cockpit/cockpit_218.bb

```diff
@@ -0,0 +1,177 @@
+SUMMARY = "Admin interface for Linux machines"
+DESCRIPTION = "Cockpit makes it easy to administer your GNU/Linux servers via a web browser"
+
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://COPYING;md5=4fbd65380cdd255951079008b364516c"
+
+SRC_URI += " \
+    https://github.com/cockpit-project/cockpit/releases/download/${PV}/cockpit-${PV}.tar.xz \
+    file://0001-remove-tests-dep-on-gobject-intro.patch \
+    file://0002-fix-makefile-use-copy-rule-for-unmodified-files.patch \
+    file://cockpit.pam \
+    "
+SRC_URI[md5sum] = "e69b0b8a75a5d55ddfd9817d857c71af"
+SRC_URI[sha256sum] = "5f242cefccd7f1120c9e0310581aa51dbe941d9c4a6d8375c45057dcbb6f2fbb"
+
+inherit gettext pkgconfig autotools systemd distro_features_check
```

Comment:
build fails on musl. I think it needs to depend on see http://errors.yoctoproject.org/Errors/Details/410052/

Reply:
- My yocto version doesn't know `features_check` yet, so I didn't make any changes there.
- Added fts in a fix
- I wanted to do a squashed force-push with the corrections. But now I added a second commit, since the first commit was picked on the master-next branch. If this is not ok, let me know.